VisualizationBase/src/Scene.h

```diff
@@ -181,16 +179,17 @@ class VISUALIZATIONBASE_API Scene : public QGraphicsScene
 		ModelRenderer* renderer_{};
 		SceneHandlerItem* sceneHandlerItem_{};
 		QList<Item*> topLevelItems_;
-		QList<ViewItem*> viewItems_;
-		ViewItem* currentViewItem_{};
 		QHash<QString, OverlayGroup> overlayGroups_;
 
+		ViewItemManager* viewItemManager_{};
+
 		Cursor* mainCursor_{};
 		bool mainCursorsJustSet_{};
 		QList<QEvent*> postEventActions_;
 
 		bool inEventHandler_{};
 		bool inAnUpdate_{};
+		friend class ViewItemManager;
```

Comment:
Please, always put friend declarations at the beginning of the `private:` section of a class.